### ISSUE TITLE
sails.js error

### ISSUE THREAD

**Issue by Wizzercn** [nexe/nexe#231]
```
[root@localhost sails]# nexe app.js 
----> input => 'app.js'
----> output => 'nexe'
----> temp => 'src'
----> framework => node
----> embedResources undefined
----> resource -> /node/sails/src/node/6.2.0/node-v6.2.0/lib/nexeres.js
----> bundle app.js
----> executing browserify via API
events.js:160
      throw er; // Unhandled 'error' event
      ^

Error: Cannot find module 'sails/node_modules/rc' from '/node/sails'
    at /soft/node-v6.2.0-linux-x64/lib/node_modules/nexe/node_modules/resolve/lib/async.js:46:17
    at process (/soft/node-v6.2.0-linux-x64/lib/node_modules/nexe/node_modules/resolve/lib/async.js:173:43)
    at ondir (/soft/node-v6.2.0-linux-x64/lib/node_modules/nexe/node_modules/resolve/lib/async.js:188:17)
    at load (/soft/node-v6.2.0-linux-x64/lib/node_modules/nexe/node_modules/resolve/lib/async.js:69:43)
    at onex (/soft/node-v6.2.0-linux-x64/lib/node_modules/nexe/node_modules/resolve/lib/async.js:92:31)
    at /soft/node-v6.2.0-linux-x64/lib/node_modules/nexe/node_modules/resolve/lib/async.js:22:47
    at FSReqWrap.oncomplete (fs.js:117:15)

```

`package.json`

``` json
"nexe": {
    "input": "app.js",
    "output": "nexe^$",
    "temp": "src",
    "runtime": {
        "framework": "node",
        "version": "6.2.0",
        "js-flags": "--use_strict",
        "ignoreFlags": true
    }
}
```

**Comment by jaredallard** [comment]
Another fun browserify bug, this is being addressed in 2.0

**Comment by calebboyd** [comment]
Closing, Please open a new issue after trying 2.0
